src/lib/casparCGState.ts

```diff
@@ -1317,10 +1318,24 @@ export class CasparCGState0 {
 						}
 
 						if (!bgDiff && newLayer.nextUp && oldLayer.nextUp && (typeof newLayer.nextUp.media !== 'string' || typeof oldLayer.nextUp.media !== 'string')) {
-							let nl = newLayer.nextUp.media
-							let ol = oldLayer.nextUp.media
+							let nMedia = newLayer.nextUp.media
+							let oMedia = oldLayer.nextUp.media
+
+							bgDiff = this.compareAttrs(nMedia, oMedia ,['inTransition','outTransition','changeTransition'])
+						}
+
+						if (!bgDiff && newLayer.nextUp && newLayer.nextUp.route && oldLayer.nextUp && oldLayer.nextUp.route) {
+							let nRoute = newLayer.nextUp.route
+							let oRoute = oldLayer.nextUp.route
+
+							bgDiff = this.compareAttrs(nRoute, oRoute, ['channel', 'layer'])
+
+							if (!bgDiff) {
+								setDefaultValue([newLayer.nextUp, oldLayer.nextUp], ['auto'], false)
+								bgDiff = this.compareAttrs(newLayer.nextUp, oldLayer.nextUp, ['auto'])
```

Comment:
Isn't this already triggering a new command in line 1317?

Reply:
I think 1317 is only working when content === MEDIA? I suppose it might make sense to just break the auto check out and have it for all content types.

Reply:
I guess channelLayout as well, since that's supported on MEDIA, INPUT, HTMLPAGE and ROUTE, correct?

Reply:
Yes, good point! Looks all good to me now.